partner_event/models/res_partner.py

```diff
@@ -0,0 +1,56 @@
+
+# © 2014 Tecnativa S.L. - Pedro M. Baeza
+# © 2015 Tecnativa S.L. - Javier Iniesta
+# © 2016 Tecnativa S.L. - Antonio Espinosa
+# © 2016 Tecnativa S.L. - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    registrations = fields.One2many(
+        string="Event registrations",
+        comodel_name='event.registration', inverse_name="attendee_partner_id")
+    event_count = fields.Integer(
+        string='Events',
+        compute='_compute_event_count',
+        help="Count of events with confirmed registrations.",
+    )
+    registration_count = fields.Integer(
+        string='Event registrations number',
+        compute='_compute_registration_count',
+        store=True)
```

Comment:
Do not store it.

Compute methods only get triggered when field is changed from this model, but creating or deleting registrations is mostly done by direct use of the `event.registration` model, so it wouldn't get updated.

The test is easy to do: just go add a registration for a partner from the events module, then go back to the partner form and see the count is wrong.